mppsolar/ports/__init__.py

```diff
@@ -19,27 +22,18 @@ class PortType(Enum):
 log = logging.getLogger("ports")
 
 
-def get_port(config):
-    log.info(f"Geting port for config '{config}'")
-    porttype = config.get("type", None)
-
+def get_port(portType, portPath, baud, protocol):
     # return None if port type is not defined
-    if porttype is None:
+    if portType is None:
         return None
 
-    # transform porttype for module lookup
-    porttype_id = f"{porttype.lower()}port"
-    # Try to import the porttype module with the supplied name (may not exist)
-    try:
-        port_module = importlib.import_module("mppsolar.ports." + porttype_id, ".")
```

Comment:
These dynamic library imports were tough to follow. Replaced with regular imports.

Reply:
Means import all modules and there are large variations